src/main/kotlin/com/vk/kphpstorm/intentions/AddImmutableClassAnnotationIntention.kt

```diff
@@ -22,6 +22,10 @@ class AddImmutableClassAnnotationIntention : PsiElementBaseIntentionAction() {
         }
 
         val klass = element.parent as PhpClass
+        if (klass.isAbstract || klass.isInterface) {
```

Comment:
add another restriction on `trait` and `anonymous classes`

Reply:
Probably, `anonymous classes` case here is extra: as far as I understand php syntax, anonymous classes aren't allowed to have names